### PR TITLE
Customized virtual assistant template for MS Teams

### PR DESCRIPTION
**Pull request by nebhagat**
Added Teams focused customized virtual assistant template.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,8 @@
 
 |    | Sample Name        | Description                                                                                                                | C#    | TypeScript   
 |:--:|:-------------------|:---------------------------------------------------------------------------------------------------------------------------|:--------|:-------------|
-|1|Hello World            | Microsoft Teams hello world sample app.                                          |[View][app-hello-world#cs]     |[View][app-hello-world#ts]
+|1|Hello World            | Microsoft Teams hello world sample app.                                           |[View][app-hello-world#cs]     |[View][app-hello-world#ts]
+|1|Teams Virtual Assistant| Customized virtual assistant template to support teams capabilities.              |[View][msteams-virtual-assistant#cs]|
```
Comment (Wajeed-msft) [inline README.md:12]: Could you please move this to [Additional Sample](https://github.com/OfficeDev/Microsoft-Teams-Samples#additional-samples)? Also please correct the hyperlink.

File: samples/msteams-virtual-assistant/csharp/.filenesting.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (Wajeed-msft) [inline samples/msteams-virtual-assistant/csharp/.filenesting.json:1]: Could you please rename this folder to match other samples? Could you please change it to bot-virtual-assistant?

File: samples/msteams-virtual-assistant/csharp/readme.md [hunk 3]
```diff
@@ -0,0 +1 @@
+Please refer to the [Virtual Assistant Template documentation](http://aka.ms/virtualassistantdocs) for deployment and customization instructions.
```
Comment (Wajeed-msft) [inline samples/msteams-virtual-assistant/csharp/readme.md:1]: Do you have instructions to run this sample locally? Could you please add those or link to specific page where developers can find these steps?

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]

File: samples/msteams-virtual-assistant/csharp/.filenesting.json (hunk 2 above)
Comment (nebhagat) [reply samples/msteams-virtual-assistant/csharp/.filenesting.json:1]: It is a sample app demonstrating how virtual assistant can be configured to use msteams platform capabilities. It has a bot, task module and compose extension. Hence don't want to name it bot specific.

File: README.md (hunk 1 above)
Comment (nebhagat) [reply README.md]: updated

File: samples/msteams-virtual-assistant/csharp/readme.md (hunk 3 above)
Comment (nebhagat) [reply samples/msteams-virtual-assistant/csharp/readme.md]: updated the link

File: README.md (hunk 1 above)
Comment (Wajeed-msft) [reply README.md]: Could you please update following? 
Sequence of sample - In table auto numbering doesn't work.
Hyperlink - I still see [View][msteams-virtual-assistant#cs] in Read me file.

File: samples/msteams-virtual-assistant/csharp/.filenesting.json (hunk 2 above)
Comment (Wajeed-msft) [reply samples/msteams-virtual-assistant/csharp/.filenesting.json:1]: Sure. Could you please then change it to app-virtual-assistant? Please change the hyperlinks present in root readme file as well.

**Review by Wajeed-msft — APPROVED** [review]

**Review by aosolis — APPROVED** [review]

**Pull request merged**